src/js/components/SharedComponents/CalendarRangeDatePicker.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { PropTypes } from 'react';
+import DayPicker, { DateUtils } from 'react-day-picker';
+
+import Moment from 'moment';
+import Navbar from './CalendarRangeDatePickerNavbar';
+
+const propTypes = {
+  numberOfMonths: PropTypes.number,
+  onSelect: PropTypes.func.isRequired,
+  minmaxDates: PropTypes.object
+};
+
+const defaultProps = {
+  numberOfMonths: 2,
+  minmaxDates: {
+      maxDate: '',
+      minDate: ''
+  }
+};
+
+export default class CalendarRangeDatePicker extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.dateValues = {};
+
+    this.state = {
+      dropdownopen: false,
+      from: null,
+      to: null
+    };
+
+    this.onDropdownChange = this.onDropdownChange.bind(this);
+    this.handleDayClick = this.handleDayClick.bind(this);
+    this.sendToFilters = this.sendToFilters.bind(this);
+  }
+
+  onDropdownChange() {
+    const currentState = this.state.dropdownopen;
+    this.setState({
+      dropdownopen: !currentState
+    });
+  }
+
+  sendToFilters() {
+    if (this.dateValues.to) {
+      const dates = {
+        startDate: Moment(this.dateValues.from).format('MM/DD/YYYY'),
+        endDate: Moment(this.dateValues.to).format('MM/DD/YYYY')
+      };
+      this.props.onSelect(dates);
+      this.setState({
+        dropdownopen: false,
+        from: '',
+        to: ''
+      });
+    }
+  }
+
+  handleDayClick(day) {
+    const range = DateUtils.addDayToRange(day, this.state);
+    this.setState(range);
+  }
+
+  drawDatePicker() {
+      this.dateValues = this.state;
+      const { from, to } = this.state;
+      const modifiers = { start: from, end: to };
+      if (this.props.minmaxDates.maxDate && this.props.minmaxDates.minDate) {
+          return (<DayPicker
+              showOutsideDays
+              fixedWeeks
+              fromMonth={this.props.minmaxDates.minDate}
+              toMonth={this.props.minmaxDates.maxDate}
+              navbarElement={<Navbar />}
+              className="innerCalendarDatePicker"
+              numberOfMonths={this.props.numberOfMonths}
+              selectedDays={[from, { from, to }]}
```

Comment:
`DayPicker` is expecting `selectedDays` to be a function rather than an array. See use of `DayPicker` [here](https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/dev/src/js/components/generateFiles/components/DatePicker.jsx#L238).

Reply:
The `DayPicker` library documentation shows that `selectedDays` can also be an array. 
Example: http://react-day-picker.js.org/examples/selected/

Reply:
Resolved by updating react-day-picker from v 2.2.3 to v 7.2.4